sympy/printing/mathml.py

```diff
@@ -1555,10 +1555,12 @@ def _print_Indexed(self, e):
     def _print_MatrixElement(self, e):
         x = self.dom.createElement('msub')
         x.appendChild(self._print(e.parent))
```

Comment:
I would suggest using the paranthesize function as stated in the other comment. (There may be cases where it is needed.)

Reply:
I did as you suggested although I am not very sure of what it did, So please have a look at it.